CHANGELOG.md

```diff
@@ -1,9 +1,70 @@
 # Mypy Release Notes
 
-## Next release
+## Mypy 1.9
+
+We’ve just uploaded mypy 1.8 to the Python Package Index ([PyPI](https://pypi.org/project/mypy/)). Mypy is a static type checker for Python. This release includes new features, performance improvements and bug fixes. You can install it as follows:
+
+    python3 -m pip install -U mypy
+
+You can read the full documentation for this release on [Read the Docs](http://mypy.readthedocs.io).
+
+#### Breaking Changes
+
+Because the version of typeshed we use in mypy 1.9 doesn't support 3.7, neither does mypy 1.9. (Marc Mueller, PR [16900](https://github.com/python/mypy/pull/16900))
+
+#### Documentation Updates
+ * Document --enable-incomplete-feature possible values in "mypy --help" (Froger David, PR [16661](https://github.com/python/mypy/pull/16661))
+ * Update new type system discussion links (thomaswhaley, PR [16841](https://github.com/python/mypy/pull/16841))
+ * Docs: Add missing class instantiation to cheat sheet (Aleksi Tarvainen, PR [16817](https://github.com/python/mypy/pull/16817))
+ * Fix typo in getting_started.rst (zipperer, PR [16700](https://github.com/python/mypy/pull/16700))
+ * Document how evil `--no-strict-optional` is (Shantanu, PR [16731](https://github.com/python/mypy/pull/16731))
+ * Improve mypy daemon documentation note about local partial types (Makonnen Makonnen, PR [16782](https://github.com/python/mypy/pull/16782))
+ * Fix numbering error in docs (Stefanie Molin, PR [16838](https://github.com/python/mypy/pull/16838))
+ * Various docs improvements (Shantanu, PR [16836](https://github.com/python/mypy/pull/16836))
+
+#### Stubtest Improvements
+  * Stubtest will ignore private function/method parameters when they are missing from the stub.
```

Comment:
```suggestion
 * Stubtest will ignore private function/method parameters when they are missing from the stub.
```
Looks misindented, not sure how it would be rendered though